scripts/anonymize.sql

```diff
@@ -47,11 +48,14 @@ TRUNCATE tidings_watch;
 
 UPDATE auth_user SET
     -- username left alone, because it's public info
-    password = NULL,
+    password = '!',
     email = CONCAT('user-', id, '@example.com'),
     first_name = ROUND(RAND()*1000000),
     last_name = ROUND(RAND()*1000000);
 
+UPDATE account_emailaddress SET
+    email = CONCAT('user-', id, '@example.com');
```

Comment:
The problem is that I'm not sure how allauth will react to the data integrity broken between the account_emailaddress and auth_user tables. If we'd use a simple hashing function on the full email address plus a random identifier that is the same for both update statements that should prevent rainbow table attacks since that secret is unknown. Something like:

```
UPDATE auth_user SET
    -- username left alone, because it's public info
    password = '!',
    email = CONCAT(MD5(CONCAT(email, @common_hash_secret)), '@example.com'),
    first_name = ROUND(RAND()*1000000),
    last_name = ROUND(RAND()*1000000);
UPDATE account_emailaddress SET
	email = CONCAT(MD5(CONCAT(email, @common_hash_secret)), '@example.com');
```
untested..